org-formation/organization.yaml

```diff
@@ -248,3 +249,13 @@ Organization:
       Tags:
         Department: Platform
         Project: Infrastructure
+
+  NlpSandboxAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: NlpSandboxAccount
+      RootEmail: aws-nlpsandbox@sagebase.org
```

Comment:
@zaro0508 .

What is the purpose of this root email?

Reply:
it's essentially the root user email.  it's for when AWS wants to send you notifications regarding the account.  it's a group mailing list and we should probably put you in the group.

Reply:
@tschaffter created an email for the group.  If you think this PR looks good, please merge.

Reply:
@thomasyu888 @zaro0508 Is it actually a good idea to use an email address that has been created as part of an AWS? If our AWS org is lost (e.g. hacked), AWS would no longer be able to contact us. Therefore, shall we specify a Google (group) address instead?

Reply:
We should use a group in sagebase.org IMO. IT admins should be the only ones with access to the root AWS account, probably for all AWS accounts in the organization.

Reply:
Thanks @ahayden, can the email: `aws-nlpsandbox@sagebase.org` be created?